src/services/FirebaseAdminService.ts

```diff
@@ -0,0 +1,20 @@
+import admin from 'firebase-admin';
+import '@firebase/auth';
+
+export const createNewFirebaseUser = async (
+  id: number,
+  email: string,
+  password: string
+): Promise<admin.auth.UserRecord | undefined> => {
```

Comment:
To blackbox this service properly I don't think we should return UserRecord. how about just returning the id of the created user?

Reply:
Yep, sounds good